src/main/java/com/mapzen/valhalla/Route.kt

```diff
@@ -1,20 +1,48 @@
 package com.mapzen.valhalla
 
 import android.location.Location
-import com.f2prateek.ln.Ln
 import com.mapzen.helpers.GeometryHelper.getBearing
 import org.json.JSONArray
 import org.json.JSONObject
 import java.lang.Math.toRadians
-import java.util.ArrayList
-import java.util.HashSet
+import java.util.*
 
 public open class Route {
+
+    private val TAG = Route::class.java.simpleName
+
     public val SNAP_PROVIDER: String = "snap"
+    /**
+     * Distance in meters
+     */
+    public val CLOSE_TO_DESTINATION_THRESHOLD: Int = 20
```

Comment:
Nice refactors. Much more readable now.

What do you think about representing distance in the actual name of the field instead of KDoc (or both)?

For example:
```kotlin
public val CLOSE_TO_DESTINATION_THRESHOLD_IN_METERS: Int = 20
```